backend/spec/factories/courses.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryBot.define do
   factory :course do
-    name { 'MyString' }
-    user
+    name { Faker::Hacker.say_something_smart }
```

Comment:
Can this method return a string length more than we expect in the model?

Reply:
No. It never return more than 256 symbols.